### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by Loupeznik**
**When merged this pull request will:**
- Add full Czech translation

### CODE REVIEW

File: addons/strings/stringtable.xml [hunk 1]
```diff
@@ -31,12 +32,8 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
+            <Czech>,</Czech>
             <German>.</German>
-            <Polish> </Polish>
```
Comment (PabstMirror) [inline addons/strings/stringtable.xml]: don't want to drop these

Comment (Loupeznik) [reply addons/strings/stringtable.xml]: Not quite sure how they got removed in the first place, I wasn't doing anything with those. Maybe some Tabler issue, I'll revise the XMLs manually.

Comment (commy2) [reply addons/strings/stringtable.xml]: Probsbly tabler cleaning white space at EOL and then the entries being empty so they get removed.

**Comment by commy2** [comment]
This is done?

**Comment by Loupeznik** [comment]
Yeah, it should be complete

**Comment by bux** [comment]
i love pabst's subtle humour

**Comment by commy2** [comment]
Thank you!

**Pull request merged**